pkg/registry/core/event/strategy.go

```diff
@@ -113,7 +113,7 @@ func ToSelectableFields(event *api.Event) fields.Set {
 		"involvedObject.resourceVersion": event.InvolvedObject.ResourceVersion,
 		"involvedObject.fieldPath":       event.InvolvedObject.FieldPath,
 		"reason":                         event.Reason,
-		"source":                         event.Source.Component,
+		"source":                         event.ReportingController,
```

Comment:
don't change the core/v1 selectable fields

Reply:
Done.